src/services/models/MediaType.ts

```diff
@@ -4,9 +4,12 @@ import type { OpenAPIMediaType } from '../../types';
 import type { RedocNormalizedOptions } from '../RedocNormalizedOptions';
 import { SchemaModel } from './Schema';
 
```

Comment:
Should we avoid line break between imports?

Reply:
I am following Redoc's linting rules and conventions set out by prettier, it doesn't seem to complain about this.